src/components/ReportActionItem/IOUAction.js

```diff
@@ -93,21 +98,21 @@ const IOUAction = (props) => {
                 action={props.action}
                 chatReportID={props.chatReportID}
                 contextMenuAnchor={props.contextMenuAnchor}
-                shouldAllowViewDetails={Boolean(props.action.originalMessage.IOUReportID)}
-                onViewDetailsPressed={launchDetailsModal}
+                onViewDetailsPressed={onIOUPreviewPressed}
                 checkIfContextMenuActive={props.checkIfContextMenuActive}
                 isHovered={props.isHovered}
             />
             {shouldShowIOUPreview && (
                 <IOUPreview
-                    iouReportID={props.action.originalMessage.IOUReportID.toString()}
+                    iouReportID={props.requestReportID}
                     chatReportID={props.chatReportID}
+                    isBillSplit={hasMultipleParticipants}
```

Comment:
Coming from https://github.com/Expensify/App/issues/18671.
We were using participants.length to determine if action is a split action, so the check would fail if you delete all of the participants from the workspace.
This was fixed by checking the original type of the action
`const isSplitBillAction = lodashGet(props.action, 'originalMessage.type', '') === CONST.IOU.REPORT_ACTION_TYPE.SPLIT;`